README.md

```diff
@@ -62,7 +62,6 @@ Preparation is covered in [Building Docker](docs/BUILDING_DOCKER.md).
 
 The build process has the following package requirements:
 
-* make
```

Comment:
Removed `make` from the list of requirements because `build-essential` takes care of it already so it is redundant.